orchagent/acltable.h

```diff
@@ -1,14 +1,16 @@
-#ifndef SWSS_ACLTABLE_H
-#define SWSS_ACLTABLE_H
+#pragma once
 
 extern "C" {
 #include "sai.h"
 }
 
+#include <map>
+#include <memory>
 #include <set>
 #include <string>
 #include <vector>
-#include <map>
+
+#include "observer.h"
 
 using namespace std;
```

Comment:
Can you remove this from the header like we've started doing with the other classes in OA?